hooks/pre-command

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+aws_account_id="697149045717"
+hash=$(echo -n "$(echo "${BUILDKITE_REPO}" | awk -F'[:.]' '{ printf $3 }')/${BUILDKITE_PIPELINE_SLUG}" | sha256sum | cut -c1-55)
+aws_role_arn="arn:aws:iam::${aws_account_id}:role/bk-${hash}-role"
+duration_seconds=${BUILDKITE_PLUGIN_OBLT_AWS_AUTH_DURATION:-3600}
+
+echo "~~~ :buildkite: Requesting OIDC token from Buildkite"
+TMPDIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'oblt-aws-auth-buildkite-plugin')
+buildkite-agent oidc request-token --audience sts.amazonaws.com > "$TMPDIR"/token.txt
```

Comment:
I don't know if the token is something valid, but do you think we can use an in-mem variable so we don't need to delete the file after using it? 

```suggestion
TOKEN=$(buildkite-agent oidc request-token --audience sts.amazonaws.com)
```

Reply:
https://github.com/elastic/oblt-aws-auth-buildkite-plugin/pull/1/commits/c3b0fef46f2302b270a98021f827985f189fb7d4